src/main/java/org/elasticsearch/client/IndicesAdminClient.java

```diff
@@ -588,6 +588,21 @@
     AnalyzeRequestBuilder prepareAnalyze(String text);
 
     /**
+     * Analyze texts under the provided index.
+     *
+     * @param index The index name
+     * @param text  The array of text to analyze
+     */
+    AnalyzeRequestBuilder prepareAnalyzeWithIndexAndMultiValued(@Nullable String index, String... text);
+
+    /**
+     * Analyze texts.
+     *
+     * @param text The array of text to analyze
+     */
+    AnalyzeRequestBuilder prepareAnalyzeWithMultiValued(String... text);
```

Comment:
Now, AnalyzeRequestBuilder doesn't have methods related `text` parameters.
If we have `prepareAnalyze()`, we should setText(String)/setText(String...) to `AnalyzeRequestBuilder.java`